### ISSUE TITLE
5.6 BETA - NFC triggered task cold not be scheduled

### ISSUE THREAD

**Issue by atl2017** [smap-consulting/smapserver#335]
### Actual Behaviour:
Task triggered by NFC cannot be scheduled by server.

In fact Email option should be hidden (Either NFC or Email but not both)

### project: Server Updating Test
Try to assign one NFC tag to a form. I could not save after that.

**Comment by atl2017** [comment]
Hi Neil,

Apparently the situation is other. If you try to edit a task after saved, you cannot save again, or if ou move from tab to select email instead of user then you cannot save.

Take a look of this short [video](https://www.useloom.com/share/4a0294c0af214e17a2210510043e2756)

**Comment by nap2000** [comment]
So that is still happening.?

Le sam. 28 juil. 2018 14:10, Alexander Torrado <notifications@github.com> a
écrit :

**Comment by atl2017** [comment]
Yes. But is not related with NFC. It is a window behaviour. By the way, did you hide "Role" assign type?
